gradle-plugins/build.gradle

```diff
@@ -14,19 +19,17 @@ repositories {
 }
 
 dependencies {
-    implementation  group: 'gradle.plugin.org.jetbrains.intellij.plugins', name: 'gradle-intellij-plugin', version: '0.4.26'
```

Comment:
or maybe just update the version on this line: `version: '1.14.1'`

Reply:
When version was specified in the _dependencies_ block gradle was pulling from https://plugins.gradle.org/m2/gradle/plugin/org/jetbrains/intellij/plugins/gradle-intellij-plugin/ which only went up to version 0.4.26 and we need 1.14.1. When IntelliJ was specified in the _plugin_ block it was able to pull the needed version.